selfdrive/controls/controlsd.py

```diff
@@ -454,9 +454,13 @@ def state_transition(self, CS):
 
     # if stock cruise is completely disabled, then we can use our own set speed logic
     if not self.CP.pcmCruise:
-      self.v_cruise_kph = update_v_cruise(self.v_cruise_kph, CS.vEgo, CS.gasPressed, CS.buttonEvents,
-                                          self.button_timers, self.enabled, self.is_metric)
-      self.v_cruise_cluster_kph = self.v_cruise_kph
+      if CS.cruiseState.available:
+        self.v_cruise_kph = update_v_cruise(self.v_cruise_kph, CS.vEgo, CS.gasPressed, CS.buttonEvents,
+                                            self.button_timers, self.enabled, self.is_metric)
+        self.v_cruise_cluster_kph = self.v_cruise_kph
+      else:
+        self.v_cruise_kph = V_CRUISE_INITIAL
+        self.v_cruise_cluster_kph = V_CRUISE_INITIAL
```

Comment:
Try to clean this up with pcmCruise logic below, can we set it to 255 for pcmCruise cars?